### PR TITLE
Pract.07: Add 05 | Margarita Gencheva

### PR DESCRIPTION
**Pull request by Margarita-Gencheva2003**
Margarita Gencheva
6 6MI0600148

### CODE REVIEW

File: SI/Sem.07/Task05Pract7.cpp [hunk 1]
```diff
@@ -0,0 +1,54 @@
+#include <iostream>
```
Comment (GeorgiTerziev02) [inline SI/Sem.07/Task05Pract7.cpp:1]: Change cpp name and place it in the correct folder

File: SI/Sem.07/Task05Pract7.cpp [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+}
+void swapKDigit(int num1, int num2, int k){//task5-pract7
+    int lenOfNum1 = lenOfNum(num1);
+    int lenOfNum2 = lenOfNum(num2);
+    int digitsInNum1[MAX_LEN_OF_NUM];
+    int digitsInNum2[MAX_LEN_OF_NUM];
+    fillDigitsOfNumInArr(digitsInNum1, lenOfNum1, num1);
+    fillDigitsOfNumInArr(digitsInNum2, lenOfNum2, num2);
+    if(k <= lenOfNum1 && k <= lenOfNum2){
+        swap(digitsInNum1[k-1], digitsInNum2[k-1]);
```
Comment (GeorgiTerziev02) [inline SI/Sem.07/Task05Pract7.cpp:45]: Write swap function yourself

File: SI/Sem.07/Task05Pract7.cpp [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        number /= 10;
+    }
+}
+int myPowF(int num, int power){
+    int newNum = 1;
+    for(int i = 0; i < power; i++){
+       newNum *= num;
+    }
+    return newNum;
+}
```
Comment (GeorgiTerziev02) [inline SI/Sem.07/Task05Pract7.cpp:24]: No need of this func

File: SI/Sem.07/Task05Pract7.cpp [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+int fromArrOfDigitsToInt(const int * arr, int len){
+    int resultNum = 0;
+    for(int i = 0; i < len; i++){
+       resultNum += arr[i] * myPowF(10, len-1-i);
```
Comment (GeorgiTerziev02) [inline SI/Sem.07/Task05Pract7.cpp:28]: no nneed of myPowF just do resultNum *= 10 on every iteration

**Review by GeorgiTerziev02 — CHANGES_REQUESTED** [review]

File: SI/Sem.07/Task05Pract7.cpp (hunk 2 above)
Comment (Margarita-Gencheva2003) [reply SI/Sem.07/Task05Pract7.cpp:45]: I have a swap function written myself.
32 to 36 line

**Pull request closed without merging**